packages/components/textarea/src/utils/helper-functions.js

```diff
@@ -0,0 +1,11 @@
+import { NEW_LINE_CHARACTER, ROWS_MAX, ROWS_MIN } from './constants';
+
+export function getRowsInRange(value) {
+	if (value <= ROWS_MIN) return ROWS_MIN;
+	if (value >= ROWS_MAX) return ROWS_MAX;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s) in this "if" block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ACurlyBraces)